src/iqm/qiskit_iqm/iqm_job.py

```diff
@@ -167,7 +170,7 @@ def result(self) -> Result:
             self._result = self._format_iqm_results(results)
             # RemoteIQMBackend.run() populates circuit_metadata, so it may be None if method wasn't called in current
             # session. In that case retrieve circuit metadata from RunResult.metadata.request.circuits[n].metadata
```

Comment:
```suggestion
            # IQMBackend.run() populates IQMJob.circuit_metadata, so it may be None if this IQMJob was created manually from a job_id.
            # In that case retrieve circuit metadata from RunResult.metadata.request.circuits[n].metadata
```
I guess this is hat the comment is trying to say?